src/lib/idmap/sss_idmap.c

```diff
@@ -1221,7 +1225,24 @@ enum idmap_error_code sss_idmap_rev_offset_identity(struct sss_idmap_ctx *ctx,
     return IDMAP_SUCCESS;
 }
 
-struct sss_idmap_offset_murmurhash3_data offset_murmurhash3_data_default = { 0xdeadbeef };
+static char *normalize_casefold(const char *input, bool normalize,
+                                bool casefold)
+{
+    if (casefold) {
+        return (char *) utf8proc_NFKC_Casefold((const utf8proc_uint8_t *) input);
```

Comment:
This ignores `ctx::alloc_func` but I guess nothing can be done here.

Reply:
Imo, this is not needed, because the alloc_func should be used for items returned to the user or which are long living. But here the value is only used by the caller for a short time and it is freed by the caller.